### DISCUSSION TITLE
Disable CLI compilation

### DISCUSSION THREAD

**Discussion by aufarijaal** [pawelsalawa/sqlitestudio#4474] [Q&A]
How to disable the cli compilation?

the link to the wiki about disabling cli compilation is not created yet. [link](https://github.com/pawelsalawa/sqlitestudio/wiki/Disabling_CLI_compilation)

**Comment by pawelsalawa** [comment]
Just edit the `SQLiteStudio3.pro` and delete the line with `cli` in the statement:
```
SUBDIRS += \
    core \
    gui \
    cli \
    gui_app
```
so it looks like:
```
SUBDIRS += \
    core \
    gui \
    gui_app
```

**Comment by aufarijaal** [comment]
oh okay thank you very much
